colibri/doc/sphinx/source/tutorials/running_fits/running_bayesian.rst

```diff
@@ -4,3 +4,164 @@
 =============
 Bayesian Fits
 =============
+
+We will first look at an example runcard to run a Bayesian fit with the Les Houches
+parametrisation model (see :ref:`this tutorial <in_les_houches>` for details
+on the Les Houches model and how to build it).
+
+Then we will look at the command to execute the runcard. 
+
```

Comment:
I think that at the top of this section there should be a discussion on what a Bayesian fit is as well as a list (eg enumeration) of what the current packages supporting bayesian sampling in colibri are (currently only nested sampling trough ultranest, but there will soon be other options too).

The subsection then focuses on a specific fit which uses ultranest / nested sampling.
The structure should allow for an easy addition of other tutorials showing how to perform sampling with other methods too.